codestodians/komlie-seller
# [Chore] Create heroku app

#### What does this PR do?
- Lays the groundwork for Heroku auto-deployment
- Fixes security vulnerabilities due to the `lodash` package.

#### How should this be manually tested?
- Go to the `komlie-seller-staging` app on Heroku.
- Deploy the `chore/create-heroku-app` branch
- View the deployed app in the browser with the given url. Everything should work correctly.

#### What are the relevant pivotal tracker stories?
- [#167339584](https://www.pivotaltracker.com/story/show/167339584)
